packages/chakra-ui/src/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -13,6 +13,7 @@ import {
 
 import { Field } from '../components/ui/field';
 import { getChakra } from '../utils';
+import { X } from 'lucide-react';
```

Comment:
Can you move this up just below the `@rjsf/utils` import. We like to keep the local imports separate from the library imports.